db/migrate/20171205205934_update_null_source_type_on_permission_templates.rb

```diff
@@ -0,0 +1,8 @@
+class UpdateNullSourceTypeOnPermissionTemplates < ActiveRecord::Migration[5.0]
+  def up
+    Hyrax::PermissionTemplate.find_each do |permission_template|
+      permission_template.source_type = 'admin_set' if permission_template.source_type.nil?
```

Comment:
Will `#find_each` iterate over every `Hyrax::PermissionTemplate` row in the database (in batches, sure)? If so, can this be simplified to:

```ruby
Hyrax::PermissionTemplate.where(source_type: nil) do |permission_template|
  permission_template.update(source_type: 'admin_set')
end
```

:question:

Reply:
This would be an improvement, and I can do so if required. My preference would be to just merge as is for the sake of time, but if you think this is important enough I can find time to change later this week.